lib/pfconfig/cached_hash.pm

```diff
@@ -177,6 +177,46 @@ sub EXISTS {
     return $self->_get_from_socket( $self->{_namespace}, "key_exists", ( search => $key ) )->{result};
 }
 
+=head2 values
+
+Added method that can be called on the underlying object of the tied hash
+Will return all the values of the hash. Mostly for internal use
+Call it using tied(%hash)->values
+
+=cut
+
+sub values {
```

Comment:
You are right also I realized sending all values could be very expensive as the config size grows.
And since pfconfig is only a single process at the moment it will block all other processes